### ISSUE TITLE
The viewport is bit out

### ISSUE THREAD

**Issue by iamaseem** [Leapcode-Open/leapcode-frontend#14]
dashboard and other pages look bit out of viewport.
it look like zoom out or something.
i attach screenshot with this.
(sorry for bad description of the issue)

**Comment by iamaseem** [comment]
![leapoutbit](https://user-images.githubusercontent.com/48835859/101055850-59150c80-35b0-11eb-91bc-99420b1dcb08.png)
![withbar](https://user-images.githubusercontent.com/48835859/101055862-5ca89380-35b0-11eb-9b04-9bd91ca1cb98.png)

**Comment by atulsuresh** [comment]
Thanks for bringing it up @iamaseem didn't notice this problem yet. Would you like to fix them?

**Comment by GJ-16** [comment]
Can I **work** on this? Can you please explain me what _has to be done_? @atulsuresh

**Comment by atulsuresh** [comment]
@iamaseem Can you please help us replicate this problem? 

We are not able to see this issue on our screens.

@GJ-16 if you noticed the same issue on the platform, happy to assign it to you.

**Comment by anujjoshi63** [comment]
@atulsuresh I can see this issue on my screen. I would like to work on it.

**Comment by adnankasem** [comment]
Hello, can anyone here give me an invite code to leapcode? I would like to start contributing to some open-source projects.

**Comment by harsh-9389** [comment]
yes, I can fix it. Can anyone tell me where I can get the source code of it?

**Comment by journeytoday** [comment]
Hello, can anyone here give me an invite code to leapcode? I would like to start contributing to some open-source projects.

**Comment by hammadahamed** [comment]
Hey guys !.,
is this already done, or assigned ?
